.gitignore

```diff
@@ -0,0 +1,40 @@
+db.properties
```

Comment:
You aren't going to be using `db.properties` so this can be removed

Reply:
Done